packages/commonwealth/server/routing/router.ts

```diff
@@ -66,7 +66,6 @@ import getUploadSignature from '../routes/getUploadSignature';
 
 import bulkOffchain from '../routes/bulkOffchain';
 import logout from '../routes/logout';
-import sendFeedback from '../routes/sendFeedback';
```

Comment:
if you are removing the whole flow, please also remove backend `sendFeedback` file. How about `SLACK_FEEDBACK_WEBHOOK` env var and slack integration?